### PR TITLE
Add Math Rendering

### PR DESCRIPTION
**Pull request by KorigamiK**
denoland/deno-gfm#37 was finally merged into main.

This PR adds a new frontmatter configuration option `render_math` to add the Katex CSS and allow rendering math using Katex.

A new 5th example post describes the method to do that

### CODE REVIEW

**Review by lino-levan — APPROVED** [review]
LGTM. Nice! @hashrock can you please take a look?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .DS_Store
+deno.lock
```
Comment (hashrock) [inline .gitignore:2]: maybe this change is not related to this PR?

**Comment by hashrock** [comment]
it would be helpful if this PR contains a test for rendering math.

**Comment by andreubotella** [comment]
Since currently all browsers (since Chrome 109) support MathML rendering natively, shouldn't Katex be configured to output MathML by default?

**Comment by lino-levan** [comment]
I think that's more of a question for https://github.com/denoland/deno-gfm

File: .gitignore (hunk 1 above)
Comment (KorigamiK) [reply .gitignore:2]: This is because deno-gfm uses esm.sh for it's dependencies which can become outdated and cause this issue denoland/deno-gfm#47

**Comment by KorigamiK** [comment]
> it would be helpful if this PR contains a test for rendering math.

It would be difficult to write a test for the math output but the 4th post shows how it can be used in the example.

**Comment by lino-levan** [comment]
I think a test would be pretty simply to write but maybe I'm not understanding some concern here. Something like

image

would be fantastic!

File: .gitignore (hunk 1 above)
Comment (hashrock) [reply .gitignore:2]: Hmm true. So we might want to "lock": false in deno.jsonc instead.

Comment (KorigamiK) [reply .gitignore:2]: Interesting, I did not know of that option in `deno.jsonc`

**Comment by hashrock** [comment]
The update to gfm@0.2.1 seems to have changed the test results.

> "$100, $200, $300, $400, $500"

into:

```
<span><span>100,100, </span><span><span><span></span><span>100</span><span>,</span></span></span></span>200, <span><span>300,300, </span><span><span><span></span><span>300</span><span>,</span></span></span></span>400, $500</p>
```


Maybe regression?

**Comment by hashrock** [comment]
merged finally! thank you @KorigamiK
